Foundation/NSLog.swift

```diff
@@ -41,3 +41,11 @@ public func NSLog(_ format: String, _ args: CVarArg...) {
         NSLogv(format, $0) 
     }
 }
+
+// hook for System.out.println()
+#if os(Android)
+public var debug = {
```

Comment:
can we name this something else? debug is potentially a very generically named symbol it could be used for storing strings, bools etc that might be something completely different and stomping upon that name might be a bit confusing to developers.

I would expect that NSLog actually hooks to android_log_print so that NSLog's show up in adb